src/components/NameBox.js

```diff
@@ -0,0 +1,31 @@
+import React, {useState , useEffect} from 'react'
+
+export const NameBox = () => {
+  const [name, setName] = useState("");
+  useEffect(() => {
```

Comment:
Does this do anything as it's defined now?

Reply:
not yet. we can add it later if we want to make something happen when the component loads for the first time.